<launch>
  <!-- <node pkg="image_view" type="image_view" name="image_view" respawn="false" output="log">
    <remap from="/image" to="/ORB_SLAM/Frame"/>
    <param name="autosize" value="true"/>
  </node> -->

	<!-- <node pkg="rviz" type="rviz" name="rviz" args="-d $(find resources)/rviz/orbslam.rviz" output="log"/> -->

 	<node pkg="dso_ros" type="dso_live" name="dso_live" cwd="node" output="screen" args="calib=$(find resources)/calibration/dso_pioneer.txt gamma=$(find resources)/gamma/pcalib.txt">
    <remap from="image" to="camera/image_raw"/>
  </node>

  <include file="$(find resources)/launch/bags/bag_play.launch"/>
</launch>
